src/app/shared/material-design/material-design.module.ts

```diff
@@ -4,10 +4,16 @@ import { MatToolbarModule } from '@angular/material/toolbar';
 import { MatButtonModule } from '@angular/material/button';
 import { MatIconModule } from '@angular/material/icon';
 import { MatSidenavModule } from '@angular/material/sidenav';
+import {MatDialogModule, MatDialog, MatDialogRef} from '@angular/material/dialog';
```

Comment:
Please remove unnecessary imports and move all these material modules to be consumed from `@angular/material` namespace